recipes/universions/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "universions" %}
+{% set version = "0.1.0.dev201911021553" %}
```

Comment:
Is there a release version available?

Reply:
Not yet, is this mandatory ?
I was trying to figure out how to publish to conda-forge before releasing to check that I would be able to publish but I can do a release if that's better

Reply:
Everything on conda-forge is considered a release, there is work on enabling pre-release builds but that currently isn't here. Tests are green and recipe looks good, so make a release once ready and we will immediately merge the PR (feel free to ping `@conda-forge/staged-recipes` then) with the released version.

Reply:
ok I'll do that !
Thank you for the answer